translate/tools/pocount.py

```diff
@@ -46,6 +46,34 @@
 default_style = style_full
 
 
+class bcolors:
```

Comment:
What does "b" refer to?  Expand the name?

Also the class name should start with a Capital letter per PEP 8.